barely.go

```diff
@@ -104,6 +118,26 @@ func (bar *StatusBar) Render(writer io.Writer) error {
 	return nil
 }
 
+func graphicLength(str string) int {
+	c := 0
+	for _, r := range str {
+		if unicode.IsGraphic(r) {
+			c++
+		}
+	}
+
+	return c
+}
+
+func trimTo(str string, l int) string {
```

Comment:
I think that should be passed as user-defined callback, because it's too specific.